hawc/apps/assessment/serializers.py

```diff
@@ -64,7 +64,7 @@ class AssessmentRootedSerializer(serializers.ModelSerializer):
     NO_PARENT = -1
 
     def get_parent(self, assessment_id, validated_data, canSelectRoot):
-        parent_id = validated_data.pop("parent")
+        parent_id = validated_data.pop("parent", self.NO_PARENT)
```

Comment:
There was an error if no `parent` was given; if you don't think this is a suitable default, we can try to make it required instead.

Reply:
I changed to make required; we'll always need to have a parent